src/main/scala/analysis.scala

```diff
@@ -165,11 +165,24 @@ case object analysis {
       override
       val sqsInitialTimeout =
         10.hours
+
+      override
+      val terminationConfig = TerminationConfig(
+        // if true loquat will terminate after solving all initial tasks
+        terminateAfterInitialDataMappings = true,
+        // if true loquat will terminate after errorQueue will contain more
+        // unique messages than threshold
+        errorsThreshold = Some(5),
```

Comment:
Be careful with this. because it's kind of inconvenient to debug problems: you get 5 failures and the whole thing goes down. Now to debug the problem you need to dig into the worker logs copied to S3 (if it didn't fail to copy it before termination). 

Definitely don't set it to 0, you're right that there may occur some occasional non-fatal issues.

Reply:
Yeah, you're right; I'll set it to None for now, it will be much safer.